src/main/java/com/intuit/graphql/authorization/rules/AuthorizationHolderFactory.java

```diff
@@ -44,6 +48,12 @@ public Map<String, Map<String, Set<String>>> parse(
         }
       }
 
+      if (invalidFieldsCollector.hasInvalidFields()) {
+        Set<FieldCoordinates> invalidFields = invalidFieldsCollector.getInvalidFields();
+        log.error(String.format("Invalid fields found in query rule.  clientId=%s, invalidFields=%s",
+          id, FieldCoordinatesFormattingUtil.toString(invalidFields)));
```

Comment:
Instead of utils, should be add a method in `invalidFieldsCollector.getInvalidFieldsAsString()`

Reply:
I'm still leaning on utils I think converting list of FieldCoordinates to a String format should be a shared one.  But I like the idea of having `invalidFieldsCollector.getInvalidFieldsAsString()` maybe use the utils in this method?

Reply:
this is resolved, implemented invalidFieldsCollector.getInvalidFieldsAsString()